nursery/check-mutex-and-exit.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: check mutex and exit
+    namespace: host-interaction/mutex
+    author: "@_re_fox"
+    scope: function
+    examples:
+      - 1d8fd13c890060464019c0f07b928b1a:0x00402eb0
```

Comment:
would you add this file to the [capa-testfiles](https://github.com/fireeye/capa-testfiles/) repository? this way our linter can ensure the rule matches as expected.

Reply:
Created PR https://github.com/fireeye/capa-testfiles/pull/3